libs/mock-block-dock/dev/dev-app.tsx

```diff
@@ -158,6 +158,7 @@ const DevApp: FunctionComponent = () => {
           initialEntities: Object.values(blockEntityMap),
         }}
         temporal={false}
+        simulateLatency={{ min: 50, max: 200 }}
```

Comment:
@nonparibus, threading this to allow for inline comments.

> Shall we add an in-UI toggle for this? 

There are quite a few props that I think would be nice to configure through the UI, but it's a bit unclear to me when we do which at the moment. I haven't looked into it but it might be that at the moment they are either in the UI _or_ they are configurable through the caller of MBD (e.g. the Hub) rather than both. @CiaranMn is that the case? If it's in the UI I think it disappears in preview mode right, and therefore is managed a different way and doesn't go through the `MockBlockDock` component props.

> Fine to dump in top-right for now as "Simulate latency", and we can clean up later.

I think we'd definitely want the ranges to be configurable so it'd involve slightly more work than just a toggle.

If we want this I'd suggest a follow-up

Reply:
`readonly` is an example of a property which can be set via both prop and via in-UI toggle. "Preview mode" just controls whether the debug panel at the bottom appears or not, it doesn't affect the value of any properties set via it.

I think it's fine to wait, since we have a sensible default that there shouldn't be any reason to mess with often.